cmd/users/main.go

```diff
@@ -172,8 +177,16 @@ func main() {
 		return
 	}
 
+	googleConfig := oauth2.Config{}
```

Comment:
Use generic names, so we can easily replace the implementation later.

Reply:
This is google specific config because we have prefixed it with `MG_GOOGLE_`

Reply:
This is Oauth2 config, it is not necessarily google-specific. The only specifc thing should be the implementation we have in package with `googleoauth` alias.

Reply:
For example, if we add GitHub we will have 2 config files with different prefixes because they have different `clientIDs` and `clientSecrets`

Reply:
If we go with multiple providers, we will use a slice/array anyway.